src/modules/logicModule/LM_PC_KPIRewarder_v1.sol

```diff
@@ -66,24 +66,28 @@ contract LM_PC_KPIRewarder_v1 is
     // This module enable KPI based reward distribution into the staking manager by using UMAs Optimistic Oracle.
 
     // It works in the following way:
-    // - The admin can create KPIs, which are a set of tranches with rewards assigned. These can be continuous or not (see below)
-    // - An external actor with the ASSERTER role can trigger the posting of an assertion to the UMA Oracle, specifying the value to be asserted and the KPI to use for the reward distrbution in case it resolves
-    // - To ensure fairness, all new staking requests are queued until the next KPI assertion is resolved. They will be added before posting the next assertion.
-    // - Once the assertion resolves, the UMA oracle triggers the assertionResolvedCallback() function. This will calculate the final reward value and distribute it to the stakers.
+    // - The admin can create KPIs, which are a set of tranches with rewards assigned. These can be continuous
+    //   or not (see below)
+    // - An external actor with the ASSERTER role can trigger the posting of an assertion to the UMA Oracle, specifying
+    //      the value to be asserted and the KPI to use for the reward distrbution in case it resolves
```

Comment:
The lines below the `-` look like they are a bit too far to the right? might just be me. first one ("The Admin can...") looks good for example.